Tools/WinMLRunner/src/EventTraceHelper.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <iostream>
+#include <windows.h>
+#include <evntrace.h>
+#include <evntcons.h>
+#include <tchar.h>
+#include <tdh.h>
+#include "ThreadPool.h"
+#include <in6addr.h>
+#include "CommandLineArgs.h"
+
+using namespace std;
+
+class EventTraceHelper
+{
+private:
+    TRACEHANDLE traceHandle;

```

Comment:
>traceHandle [](http://example.com/codeflow?start=16&length=11)

It is nice to have member var prefixed with _ so that the members are clearly visible in the implementation.